sdk/cosmos/azure-cosmos/azure/cosmos/aio/user.py

```diff
@@ -115,23 +118,24 @@ def list_permissions(self, max_item_count=None, **kwargs):
     @distributed_trace
     def query_permissions(
         self,
-        query,
-        parameters=None,
-        max_item_count=None,
-        **kwargs
-    ):
-        # type: (str, Optional[List[str]], Optional[int], Any) -> AsyncItemPaged[Dict[str, Any]]
+        query: str,
+        parameters: Optional[List[Dict[str, Any]]] = None,
```

Comment:
I left parameters outside of the kwargs since it's a part of the query methods that users should be aware of and that users constantly use along with their queries

Reply:
nvm im silly